translations/monero-core_da.ts

```diff
@@ -1,2822 +1,2812 @@
-    </message>
-    <message>
-        <location filename="../pages/SharedRingDB.qml" line="267"/>
-        <source>This records rings used by outputs spent on Monero on a key reusing chain, so that the same ring may be reused to avoid privacy issues.</source>
-        <translation>Denne optager ringe der bliver brugt af outputs på Monero på en nøgle-genbrugene blockchain, så den samme ring kan blive genbrugt for at undgå anonymitets problemer.</translation>
-    </message>
-    <message>
-        <location filename="../pages/SharedRingDB.qml" line="278"/>
-        <source>Key image</source>
-        <translation>Nøgle billede</translation>
-    </message>
-    <message>
-        <location filename="../pages/SharedRingDB.qml" line="279"/>
-        <source>Paste key image</source>
-        <translation>Indsæt nøgle billede</translation>
-    </message>
-    <message>
-        <location filename="../pages/SharedRingDB.qml" line="299"/>
-        <source>Get ring</source>
-        <translation>Hent ring</translation>
-    </message>
-    <message>
-        <location filename="../pages/SharedRingDB.qml" line="311"/>
-        <source>Get Ring</source>
-        <translation>Hent Ring</translation>
-    </message>
-    <message>
-        <location filename="../pages/SharedRingDB.qml" line="317"/>
-        <source>No ring found</source>
-        <translation>Ingen ring fundet</translation>
-    </message>
-    <message>
-        <location filename="../pages/SharedRingDB.qml" line="334"/>
-        <source>Set ring</source>
-        <translation>Sæt ring</translation>
-    </message>
-    <message>
-        <location filename="../pages/SharedRingDB.qml" line="346"/>
-        <source>Set Ring</source>
-        <translation>Sæt Ring</translation>
-    </message>
-    <message>
-        <source>I intend to spend on key-reusing fork</source>
-        <translation type="vanished">Jeg har hensigt at bruge på en nøgle-genbrugene fork(s)</translation>
-    </message>
-    <message>
-        <location filename="../pages/SharedRingDB.qml" line="378"/>
-        <source>I might want to spend on key-reusing fork(s)</source>
-        <translation>Jeg har nok hensigt at bruge på en nøgle-genbrugene fork(s)</translation>
-    </message>
-    <message>
-        <location filename="../pages/SharedRingDB.qml" line="392"/>
-        <source>Relative</source>
-        <translation>Relative</translation>
-    </message>
-    <message>
-        <location filename="../pages/SharedRingDB.qml" line="408"/>
-        <source>Segregation height:</source>
-        <translation>Adskilleses højde:</translation>
-    </message>
-</context>
-<context>
-    <name>Sign</name>
-    <message>
-        <location filename="../pages/Sign.qml" line="60"/>
-        <source>Good signature</source>
-        <translation>God signatur</translation>
-    </message>
-    <me
```

Comment:
se-kun tegnebøgen --> Se-kun tegnebogen  ?

"Avbrydke" ???  Do you mean "lukke"?